python/cuml/metrics/confusion_matrix.py

```diff
@@ -52,25 +53,33 @@ def confusion_matrix(y_true, y_pred,
         Normalizes confusion matrix over the true (rows), predicted (columns)
         conditions or all the population. If None, confusion matrix will not be
         normalized.
+    convert_dtype : bool, optional (default = False)
+        When set to True, the confusion matrix method will automatically
+        convert the inputs to np.int32.
```

Comment:
To improve clarity, we should maybe slightly modify this portion of the documentation. We might want to specify that (when set to True) it converts the prediction and ground truth arguments (not labels). Or otherwise, we might want to change the code so that all inputs get converted.

Reply:
Good point! I'll explicitly state that we won't convert the labels because I don't think it makes sense to have them in `np.float`, right?

Reply:
Although thinking more, if we are anyway converting the predictions and ground truth to floating point, might as well do it for labels

Reply:
Changes look good!